src/components/templates/BottomSheetCreateRenameGroup/index.tsx

```diff
@@ -100,7 +100,7 @@ export const BottomSheetCreateRenameGroup = forwardRef<BottomSheetRef, Props>(
     return (
       <>
         <BottomSheetFloat
-          height={350}
+          height={320}
```

Comment:
We can use Math.min(verticalScale(320), 320) here to match different screen size